### DISCUSSION TITLE
Game list.

### DISCUSSION THREAD

**Discussion by Happi4Ever** [skyline-emu/skyline#1075] [Q&A] [answered]
Could someone please give me a list of games that run as of now?

**Comment by s3lverd** [comment]
Here you go https://docs.google.com/spreadsheets/d/1oFND6f0vcQKsb8dMiVuhrwY-_Tkzf2RyEbjBM64b_eM/edit#gid=990708478 thats the official one you can find on the discord too :)

**Comment by s3lverd** [comment, marked as answer]
Also Next Time Just ask in the skyline discord https://discord.gg/XnbXNQM instead of making an issue

**Reply by Happi4Ever** [reply]
Hello and thanks! Anyway sorry for making it an issue and bye.














Among us 🤨🤨🤨🤨🥶🤨🥶🥶🥶🤨💀💀
